components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/WorkflowManagementServiceImpl.java

```diff
@@ -151,10 +151,8 @@ public Workflow getWorkflow(String workflowId) throws WorkflowException {
             }
         }
         for (WorkflowListener workflowListener : workflowListenerList) {
-            if (workflowListener.isEnable()) {
-                if (workflowListener.isEnable()) {
-                    workflowListener.doPostListWorkflowEvents(eventList);
-                }
+            if (workflowListener.isEnable() && workflowListener.isEnable()) {
```

Comment:
Seems to be same condition has repeated in both if clauses unnecessarily in previous code, can u remove 1 of them?